src/extension.ts

```diff
@@ -367,14 +373,40 @@ export function deactivate() {
   outputChannel.appendLine('Ocean Protocol extension is being deactivated')
 }
 
+async function handleFailureLogsRetrieval(
+  config: SelectedConfig,
+  jobId: string,
+  status: any,
+  resultsFolderPath: string,
+  computeLogsChannel: vscode.OutputChannel,
+  progress: vscode.Progress<{ message?: string }>
+) {
+  if (!status.results || status.results.length === 0) return
```

Comment:
a humble improvement here would be to write it in one go:

```suggestion
  if (status.results?.length < 1) return
```

Reply:
w8, I was actually wrong on this one, because undefined < 1 is false 
<img width="554" height="190" alt="image" src="https://github.com/user-attachments/assets/b116dc55-6cfa-4c6e-a626-259d8ef56860" />
my bad